data/systems/custom/07_1orionis.lua

```diff
@@ -95,7 +95,7 @@ local orionisd = CustomSystemBody:new('Neart', 'PLANET_TERRESTRIAL') --Means "st
 	:ice_cover(f(5,1000))
 	:life(f(0,1))
 
-local orionisf = CustomSystemBody:new('Iona', 'PLANET_GAS_GIANT') --Real scottish island
+local orionisf = CustomSystemBody:new('Ì', 'PLANET_GAS_GIANT') --Real scottish island
```

Comment:
The rest of the placenames look good, but I'm going to have to ask that this be converted back to `Iona` - not only is the pronunciation non-obvious for those who don't speak Gaelic, but single-character names for bodies have significant negative usability impact (try finding it in worldview!) and breaks homogeneity with the rest of the bodies in the system.

Reply:
I'll use "Hiort" instead - also an island, longer and no accents :)